dicoogle/src/main/java/pt/ua/dicoogle/core/query/ExportToCSVQueryTask.java

```diff
@@ -103,11 +104,16 @@ private void printLine(SearchResult result){
 		StringBuilder builder = new StringBuilder();
 		
 		HashMap<String, Object> extraFields = result.getExtraData();
-
+		
 		for (String tag : tagsOrder) {
 			Object temp1 = extraFields.get(tag);
-
-			String s = (temp1 != null) ? StringUtils.trimToEmpty(temp1.toString()) : "";
+			String temp1String = temp1.toString();
+			
+			if(NumberUtils.isNumber(temp1String)){
+				temp1String = NumberUtils.createBigDecimal(temp1String).toPlainString();	
```

Comment:
Do we really need to print numbers without an exponent field? What if they are extremely large/small in absolute value?